### PR TITLE
Crud refactor to follow MVC pattern

### PR DESCRIPTION
**Pull request by reedbruskin**

### CODE REVIEW

File: model/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    return pool.query('INSERT INTO items(item_name, best_seller, price, price_reduction, in_stock, us_free_shipping, carts_item_is_in) VALUES($1, $2, $3, $4, $5, $6, $7)', [data.item_name, data.best_seller, data.price, data.price_reduction, data.in_stock, data.us_free_shipping, data.carts_item_is_in])
+  },
+
+  update: (itemId, data) => {
+    return pool.query('UPDATE items SET item_name = $1, best_seller = $2, price = $3, price_reduction = $4, in_stock = $5, us_free_shipping = $6, carts_item_is_in = $7 WHERE item_id = $8',
+    [data.item_name, data.best_seller, data.price, data.price_reduction, data.in_stock, data.us_free_shipping, data.carts_item_is_in, itemId])
+  },
+
+  delete: (itemId) => {
+    console.log(itemId)
```
Comment (grezzzy) [inline model/index.js:22]: Do you want this piece to stay in the production code?

File: server/index.js [hunk 2]
```diff
@@ -6,79 +6,91 @@ const express = require('express');
...
-    if (err) {
-      console.log(`failed to add new item: ${err}`);
-      res.status(404).send(err);
-    } else {
-      console.log('successfully added new item');
-      res.status(200).send(data);
-    }
-  });
-});
+  controller.shopping.create(req.body)
```
Comment (grezzzy) [inline server/index.js:32]: do you want to be sending the whole body here? or should you check for the properties you need first? i think it works either way - might just change the error handling

**Review by grezzzy — APPROVED** [review]
Nice job with MVC - looks good!

File: model/index.js (hunk 1 above)
Comment (reedbruskin) [reply model/index.js:22]: Thanks for catching that. Removed.

File: server/index.js (hunk 2 above)
Comment (reedbruskin) [reply server/index.js:32]: Req.body actually wound up being only the info I send from Postman (i.e. only the object w/ exact data I need). I might have to adjust if I wind up needing post functionality on the front end.

**Pull request merged**